### DISCUSSION TITLE
Controlling multiple arduino

### DISCUSSION THREAD

**Discussion by trker** [Drumber/RemoteLight#29] [Ideas]
Hello,
The software doesn't allow to manage more than one arduino at the same time.
If we open the same version (java java or windows windows) of the remote light software in my trial, it does not work.
allows to manage an arduino.
But it allows to manage 2 devices while one is opening java and the other is windows application.
Will you develop software to manage more than one arduino with a single software?
It would be great if you add an update like this.

**Comment by Drumber** [comment]
Hi, I'm aware of this limitation and I'm already planning for a big update that adds the ability to control multiple devices/outputs at the same time.
Right now you can only link multiple outputs together using the `Chain` or `MultiOutput`. But this will only show the same effects on all outputs.
Starting multiple RemoteLight instances can cause some errors since it was not designed to have multiple RemoteLight instances running simultaneously.
